Packs/GetListRow/Scripts/GetListRow/GetListRow.py

```diff
@@ -0,0 +1,84 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+
+def validate_args(parse_all, header, value):
+    if parse_all.lower() != 'true' and (not header or not value):
+        return_error("Error: If parse_all is false, both header and value must be specified.")
+
+
+def validate_list_exists(list_result):
+    if not list_result or "Item not found" in list_result:
+        return_error("Error: The supplied list name not found.")
+
+
+def validate_header_exists(headers, header):
+    if header not in headers:
+        return_error("Error: The supplied header name not found.")
+
+
+def list_to_headers_and_lines(list_result):
+    lines_without_headers = [line.split(',') for line in list_result.split('\n')]
```

Comment:
This is with the  headers.
lines_without_headers should start from index 1, like so: `[1:]`